### PR TITLE
add function to get latest ansys installation

### LINKED ISSUE #38: Add method to get the latest available Ansys installation
### 📝 Description of the feature

It would be useful to have a method that returns the location of the latest installed Ansys. Since ``get_available_ansys_installations`` can return both normal and Student installs, I would propose the following logic:
- Always use the latest available release
- If there is both a student and regular install for the latest release, use the regular install.

Use case: for providing a useful default when configuring how a product is launched.

### 💡 Steps for implementing the feature

Example implementation:

```python
def get_latest_ansys_installation():
    installations = get_available_ansys_installations()
    if not installations:
         raise ValueError("No Ansys installation found.")

    def sort_key(version_nr):
        if version_nr < 0:
            return abs(version_nr) - 0.5
        return version_nr

    latest_key = max(installations, key=sort_key)
    return installations[latest_key]
```

### PR DESCRIPTION
**Pull request by Gryfenfer97**
Fix #38

### CODE REVIEW

**Review by germa89 — APPROVED** [review]
LGTM. 

FYI: Students versions return a negative version, so there might be the case where this function will return the student version. But because we are asking the latest, this makes sense, although we might need to change it in the future.

**Comment by greschd** [comment]
> But because we are asking the latest, this makes sense, although we might need to change it in the future.

I would argue if both student and non-student versions are installed, we should predictably choose non-student instead of randomly picking between the two. 
That's the reason for the somewhat complicated `sort_key` outlined in #38. The code there can probably be made more legible, e.g. by using a tuple `(abs(version), version)` to sort by.

**Comment by germa89** [comment]
> > But because we are asking the latest, this makes sense, although we might need to change it in the future.
> 
> I would argue if both student and non-student versions are installed, we should predictably choose non-student instead of randomly picking between the two. That's the reason for the somewhat complicated `sort_key` outlined in #38. The code there can probably be made more legible, e.g. by using a tuple `(abs(version), version)` to sort by.

I think I didn't really appreciate the complicated sorting key in your comment. Indeed it is a good idea the "-0.5". Can @Gryfenfer97 implement that?

File: tests/test_path.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
     find_mechanical,
     get_ansys_path,
     get_available_ansys_installations,
+    get_latest_ansys_installation,
```
Comment (germa89) [inline tests/test_path.py:19]: This function is in `ansys.tools.path.path` ... ;)

Feel free to add it to the `__init__` though.

**Comment by greschd** [comment]
> I think I didn't really appreciate the complicated sorting key in your comment

Heh yeah, it was a bit cryptic. A comment might've helped 🙂

**Comment by Gryfenfer97** [comment]
> > I think I didn't really appreciate the complicated sorting key in your comment
> 
> Heh yeah, it was a bit cryptic. A comment might've helped 🙂

Indeed, that was a bit complicated

**Pull request merged**